presto-orc/src/main/java/com/facebook/presto/orc/writer/ByteColumnWriter.java

```diff
@@ -115,7 +116,9 @@ public long writeBlock(Block block)
             }
         }
         // For byte columns, null and values has the same size (1 byte)
-        return block.getPositionCount() * NULL_SIZE;
+        long rawSize = block.getPositionCount() * NULL_SIZE;
```

Comment:
ByteColumnWriter and some other column writers do not have stats builder, they create ColumnStats directly.